src/main/java/cz/student/richard808code/App.java

```diff
@@ -1,4 +1,48 @@
 package cz.student.richard808code;
 
 public class App {
+    public static void main(String[] args) {
+        // Vytvoření uživatele
+
+        User user1 = new User();
+        user1.setUserName("username1");
+        user1.setEmail("email1@gmail.com");
+        user1.setPassword("password1");
+
+        UserDao userDao = new UserDao();
+
+        //Test metody create.
+        User createdUser1 = userDao.create(user1);
+
+        System.out.println("User created with ID: " + createdUser1.getId());
+
+
+        //Test metody read pro existující záznam.
+
+        System.out.println(userDao.read(1));
```

Comment:
CLEAN-CODE: lepsi popis do konzole

```suggestion
System.out.println("Find user with id = 1: " + userDao.read(1));
```

Reply:
Opraveno